Tests/ASPagerNodeTests.m

```diff
@@ -90,6 +90,14 @@ - (void)testPagerReturnsNotFoundForCellThatDontExistInPager {
   XCTAssertEqual([testController.pagerNode indexOfPageWithNode:badNode], NSNotFound);
 }
 
+- (void)testScrollPageToIndex {
```

Comment:
Per our project convention, please put the bracket on a new line.

While on it, please also update other methods to follow the same rule.